video/rooms/participants/kick-remove-participant/remove-participant.java

```diff
@@ -0,0 +1,18 @@
+import com.twilio.Twilio;
+import com.twilio.rest.video.v1.room.Participant;
+
+public class ParticipantsExample2 {
+	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+	public static final String API_KEY_SECRET = "your_api_key_secret";
+
+	public static void main(String args[]) {
```

Comment:
Can we use "4 spaces" in all java files?  This appears to be 8 spaces..

Reply:
The source files look good in Eclipse or even Sublime. Not sure why it goes to 8 spaces here on GH. Any suggestions ? @innerverse